docs/7.x/changelog.md

```diff
@@ -54,6 +54,21 @@ extend updates past End of Support through customer agreements if required.
 
 7.1 is currently pre-release.
 
+### 7.1.0-alpha.6 (April 14, 2021)
+
+#### Improvements
+* Add GCE Alias IP support for flannel ([#2439](https://github.com/gravitational/gravity/pull/2439), [planet#830](https://github.com/gravitational/planet/pull/830), [flannel#10](https://github.com/gravitational/flannel/pull/10))
+* Improve performance of AWS autoscaling integration ([#2458](https://github.com/gravitational/gravity/pull/2458))
+* Improve satellite cluster health monitoring on large clusters up to 1000 nodes ([#2439](https://github.com/gravitational/gravity/pull/2439), [planet#828](https://github.com/gravitational/planet/pull/828), [satellite#297](https://github.com/gravitational/satellite/pull/297))
+
+#### Bugfixes
+* Fix ownership of monitoring and logrange directories ([#2448](https://github.com/gravitational/gravity/pull/2448))
+* A kubernetes config is no longer required for `gravity app sync` ([#2438](https://github.com/gravitational/gravity/pull/2438))
+
+#### Internal Changes
+* Upgrade internal kubernetes client go to v1.19.8 and helm to v3.4.2. Removed helm2. ([#2433](https://github.com/gravitational/gravity/pull/2433), [planet#827](https://github.com/gravitational/planet/pull/827), [rigging#100](https://github.com/gravitational/rigging/pull/100), [satellite#299](https://github.com/gravitational/satellite/pull/299))
```

Comment:
@bernardjkim I may have made up that helm 2 was removed.  Can you check this note for me, please?

Reply:
Helm v2 binary is still included in gravity. The integrated helm/app command has just been upgraded to use helm v3

Reply:
I still don't really know the use cases, so I'm not sure if we can just remove helm v2 binary or not.

Reply:
Helm 2 is no longer mentioned in 6aa4ac2.  My impression is without tiller, the helm 2 binary is somewhat useless.  Customers would have to run their own tiller (in which case they could probably bootstrap their own helm 2 in the hooks containers).

If we do remove helm 2, it should be before 7.1.0 releases.  It'd be bad form to remove something like that in a patch release.

Reply:
Hmm, that's a good point. I'll do some clean up before 7.1.0 release.